AnalysisTools/plugins/PATObjectCounter.cc

```diff
@@ -43,32 +43,36 @@ class PATObjectCounter : public edm::stream::EDProducer<>
   virtual void produce(edm::Event& iEvent, const edm::EventSetup& iSetup);
 
   const edm::EDGetTokenT<edm::View<T> > srcToken_;
-  const std::vector<std::string> cuts_;
+  const std::vector<std::string> cut_strings_;
```

Comment:
With apologies for being obnoxiously picky, could we put this variable in camelCase like everything else?